src/NuGet.Core/NuGet.Protocol.Core.v3/DownloadResourceV3.cs

```diff
@@ -170,13 +168,16 @@ private DownloadResourceV3(HttpSource client)
                 }
                 catch (IOException ex) when (ex.InnerException is SocketException && i < 2)
                 {
-                    string message = $"Error downloading {identity} from {uri} {ExceptionUtilities.DisplayMessage(ex)}";
-
+                    string message = string.Format(Strings.Log_ErrorDownloading, identity, uri)
```

Comment:
use string.Format(CultureInfo.CurrentCulture, Strings.   everywhere instead of the default. This kind of stuff use to get caught by FXCop but we don't have that in DNX projects.